<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Flash</class>
 <widget class="QWidget" name="Flash">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Flash</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resourcesGUI.qrc">
    <normaloff>:/tray/flash</normaloff>:/tray/flash</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>File Path</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLineEdit" name="editFilename"/>
        </item>
        <item>
         <widget class="QToolButton" name="buttonFile">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupProgress">
     <property name="title">
      <string>Progress</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="labelOperation">
        <property name="text">
         <string>Erasing</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progress">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>89</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonErase">
       <property name="text">
        <string>Mass erase</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonFlash">
       <property name="text">
        <string>Flash</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>groupBox</zorder>
  <zorder>groupProgress</zorder>
  <zorder>verticalSpacer</zorder>
 </widget>
 <resources>
  <include location="resourcesGUI.qrc"/>
 </resources>
 <connections/>
</ui>
